data/korath outfits.txt

```diff
@@ -91,6 +91,15 @@ outfit "Large Heat Shunt"
 	"cooling" 54
 	description "Without these powerful cooling systems, most Korath warships would overheat within seconds as soon as they began to fire their weapons."
 
+outfit "Fuel Collector"
+	category "Systems"
+	cost 101000
+	thumbnail "outfit/unknown"
+	"mass" 20
+	"outfit space" -20
+	"fuel capacity" 200
+	"ramscoop" 0.5
+	description "Because of their habit of raiding systems near and far, Korath ships need a considerable amount of hyperspace fuel for their jump drives. The Fuel Collector is a bulky system which has both a ramscoop to harvest additional fuel and a large capacity to store it."
```

Comment:
As a grammar nazi, I will not accept [the use of "which" here instead of "that".](http://www.quickanddirtytips.com/education/grammar/which-versus-that-0)